src/Zed/Common/Factory/StatelessFactoryRule.php

```diff
@@ -1,52 +0,0 @@
-<?php
-
-namespace ArchitectureSniffer\Zed\Common\Factory;
-
-use PHPMD\AbstractNode;
-use PHPMD\AbstractRule;
-use PHPMD\Node\ClassNode;
-use PHPMD\Rule\ClassAware;
-
-/**
- * Factories shouldn't contain any property. Properties would make the factory stateful.
- */
-class StatelessFactoryRule extends AbstractRule implements ClassAware
```

Comment:
Is that check moved to somewhere else? I think the rule is valid and shouldn't be removed.